src/App.tsx

```diff
@@ -1,61 +1,116 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+
+const preparedTodos = todosFromServer.map(todo => {
+  return {
+    ...todo,
+    user: usersFromServer.find(user => user.id === todo.userId) as User,
+  };
+});
 
 export const App = () => {
+  const [todos, setTodos] = useState<Todo[]>(preparedTodos);
+  const [title, setTitle] = useState('');
+  const [selectUserId, setSelectUserId] = useState(0);
+
+  const [titleError, setTitleError] = useState(false);
+  const [selectUserIdError, setSelectUserIdError] = useState(false);
+
+  const getMaxTodoId = (todoId: Todo[]) => {
```

Comment:
- The name is `todoId` but actually it's an array of todos 👀 
- Move this helper outside the component